group_vars/nibbler_cluster/vars.yml

```diff
@@ -49,6 +49,12 @@ ui_features: 'prm02,tmp02'
 ui_ethernet_interfaces:
   - 'eth0'
   - 'eth1'
+irods_sockets: 4
+irods_cores_per_socket: 1
+irods_real_memory: 3790
+irods_local_disk: 0
+irods_features: 'tmp02'
```

Comment:
Features are only used in `slurm.conf` and there is no tmp02 on the iRODS machines.

Reply:
well spotted :) Where not used at all.